R/categorize.R

```diff
@@ -11,6 +11,31 @@
 #' @export
 #' @examples
 #' categorize(data.frame(a = c(1.2,2.3), b = c('a','b','c')))
-categorize <- function(df) {
-    pass
-}
\ No newline at end of file
+categorize <- function(df, max_cat = 10) {
+  # Check that inputs are valid
+  if(max_cat%%1 != 0 | max_cat < 1) stop("Error: max_cat must be a positive integer")
```

Comment:
Please add a script to deal with a case in which user passes string input to the max_cat

Reply:
See commit below! Good catch!